mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -1219,6 +1224,13 @@ def LLVM_AddressOfOp : LLVM_Op<"mlir.addressof",
 
     /// Return the llvm.func operation that is referenced here.
     LLVMFuncOp getFunction(SymbolTableCollection &symbolTable);
+
+  }];
+
+  let extraClassDefinition = [{
```

Comment:
Same here, add it next to https://github.com/trailofbits/instafix-llvm/blob/f21b247d13cda3a9f844078aeacaad5bd17d3905/mlir/lib/Dialect/LLVMIR/IR/LLVMDialect.cpp#L2039